### ISSUE TITLE
Feature Request: placeholder localization

### ISSUE THREAD

**Issue by shinxi** [predixdesignsystem/px-datetime-picker#38]
Currently placeholder is hard coded via [date format](https://www.predix-ui.com/#/elements/datetime/px-datetime-field:property-dateFormat), e.g., YYYY for year, MM for month, DD for day.
![image](https://user-images.githubusercontent.com/1262296/54125146-296abc00-443f-11e9-932a-c627c5e08aa9.png) This might cause confusion for these non-technical users, it'll make more sense to make the date format to be localizable. E.g.,
```js
_placeholderText: function(momentFormat){
  ...
  return this.localize(momentFormat);
}

```
cc @evanjd @joshsylvester @shao @inothings

**Comment by joshsylvester** [comment]
this sounds like a logical change - but wouldn't the same affect all date-related fields that have placeholder text?

**Comment by shinxi** [comment]
> this sounds like a logical change - but wouldn't the same affect all date-related fields that have placeholder text?

@joshsylvester Yes, we should do the enhancement at https://github.com/predixdesignsystem/px-datetime-common

**Comment by evanjd** [comment]
Released in [v2.3.0](https://github.com/predixdesignsystem/px-datetime-picker/releases/tag/v2.3.0), closing.
